control-plane/pkg/prober/async_prober_test.go

```diff
@@ -214,9 +214,9 @@ func TestAsyncProber(t *testing.T) {
 				return status == tc.wantStatus
 			}
 
-			require.Eventuallyf(t, probeFunc, 5*time.Second, 100*time.Millisecond, "")
-			require.Eventuallyf(t, func() bool { return wantRequestCountMin.Load() == 0 }, 5*time.Second, 100*time.Millisecond, "got %d, want 0", wantRequestCountMin.Load())
-			require.Eventuallyf(t, func() bool { return wantRequeueCountMin.Load() == 0 }, 5*time.Second, 100*time.Millisecond, "got %d, want 0", wantRequeueCountMin.Load())
+			require.Eventuallyf(t, probeFunc, 5*time.Second, 250*time.Millisecond, "")
```

Comment:
Not exactly, if you look here: https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/control-plane/pkg/prober/async_prober_test.go#L155 you will see that we actually are calling `t.Run("test name", func(t *testing.T){})`, so we are just redefining `t` within the scope of the function to refer to the child test, rather than the parent.